x-pack/plugins/lens/public/embeddable/embeddable.tsx

```diff
@@ -168,7 +168,8 @@ interface LensBaseEmbeddableInput extends EmbeddableInput {
   onLoad?: (
     isLoading: boolean,
     adapters?: Partial<DefaultInspectorAdapters>,
-    output$?: Observable<LensEmbeddableOutput>
+    output$?: Observable<LensEmbeddableOutput>,
+    embeddable?: AbstractEmbeddable<LensEmbeddableInput, LensEmbeddableOutput>
   ) => void;
```

Comment:
@stratoula this should be refactored. You now only need to send the embeddable and not the adapters or the observable. Discover needs to be refactored too to accommodate this change